bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1725,7 +1725,7 @@ lnd/clm2/surfdata_map/release-clm5.0.30/surfdata_ne0np4.CONUS.ne30x8_hist_78pfts
 <stream_year_last_urbantv  sim_year="constant" sim_year_range="1850-2000" >2106</stream_year_last_urbantv>
 
 <stream_fldfilename_urbantv phys="clm5_1"  hgrid="0.9x1.25"
->lnd/clm2/urbandata/CLM50_tbuildmax_Oleson_2016_0.9x1.25_simyr1849-2106_c160923.nc</stream_fldfilename_urbantv>
+>/glade/p/cgd/tss/people/oleson/urban_sfcdata/Feddema_urban_data_080410/HIGH_RES_VERSION2/URBAN_PROPERTIES_THESIS_TOOL/JUN_22_2018/CLM50_tbuildmax_Oleson_2016_0.9x1.25_simyr1849-2106_c200605.nc</stream_fldfilename_urbantv>
```

Comment:
Comments from @ekluzek 
- Better to rename the file 
CLM52_tbuildmax_Oleson_2016_0.9x1.25_simyr1849-2106_c200605.nc
and is it still based on Oleson (2016) or a newer publication?
- After that, Sam can copy to lnd/clm2/urbandata and run `rimport` to get the file in the repository.

Reply:
Good point.  I've rename the file appropriately:

/glade/p/cgd/tss/people/oleson/urban_sfcdata/Feddema_urban_data_080410/HIGH_RES_VERSION2/URBAN_PROPERTIES_THESIS_TOOL/JUN_22_2018/CTSM52_tbuildmax_OlesonFeddema_2020_0.9x1.25_simyr1849-2106_c200605.nc

Reply:
I've updated the file name in bld/namelist_files/namelist_defaults_ctsm.xml and pushed to the branch.

Reply:
Completed this "rimport"

Reply:
@slevisconsulting this file needs to be copied over to inputdata/lnd/clm2/urbandata so that it can be automatically downloaded. Having the full path allows this to run on cheyenne, but the general operation needs it to be under the inputdata directory structure.

Reply:
@ekluzek thank you. I copied the file correctly but neglected to explicitly say so by committing & pushing the relevant change in `namelist_defaults_ctsm.xml` because I still have to copy and rimport the transient urban files. Doing that next.